admin/page/produk.php

```diff
@@ -0,0 +1,243 @@
+<script src="https://cdn.ckeditor.com/ckeditor5/19.1.1/decoupled-document/ckeditor.js"></script>
+
+
+
+<div class="mt-4 w-100">
+    <input type="search" placeholder="Search" id="search" name="search" class="search ml-auto float-right">
+</div>
+
+<table class="table" id="tabel">
+    <thead>
+        <th style="background-color: #204a87ff; text-align: center;">ID</th>
+        <th style="background-color: #204a87ff; text-align: center;">Nama Produk</th>
+        <th style="background-color: #204a87ff; text-align: center;">Jenis Produk</th>
+        <th style="background-color: #204a87ff; text-align: center;">Kategori</th>
+        <th style="background-color: #204a87ff; text-align: center;">Gambar</th>
+        <th style="background-color: #204a87ff; text-align: center;">Video</th>
+        <th style="background-color: #204a87ff; text-align: center;">Spek</th>
+        <th style="background-color: #204a87ff; text-align: center;">Deskripsi</th>
+        <th style="background-color: #204a87ff; text-align: center;">Download</th>
+    </thead>
+    <tbody id="tbody">
+        <?php
+            $select = mysqli_query($connection, "select * from produk");
+            while($data = mysqli_fetch_array($select)){
+                echo "
+                    <tr data-toggle='modal' data-target='#modelId'>
+                        <td class='text-center'>".$data["id_produk"]."</td>
+                        <td class='text-center'>".$data["nama_produk"]."</td>
+                        <td class='text-center'>".$data["jenis_produk"]."</td>
+                        <td class='text-center'>".$data["kategori"]."</td>
+                        <td class='text-center'>".$data["gambar"]."</td>
+                        <td class='text-center'>".$data["video"]."</td>
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$data["video"]'.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)